docs/migrate/migrate-subgraph.md

```diff
@@ -0,0 +1,152 @@
+
+## Squid setup 
+
+### 1. Clone template
+
+To begin with, clone the `ethereum-minimal` template from the [squid-evm-template repo](https://github.com/subsquid/squid-evm-template):
+
+```bash
+git clone -b ethereum-minimal https://github.com/subsquid/squid-evm-template.git
+cd squid-evm-template
+```
+
+### 2. Copy the schema file and generate entities
+
+The minimal template already contains a dummy `schema.graphql` file. We replace it with the subgraph's schema as is:
+
+```gql file=schema.graphl
+type Gravatar @entity {
+  id: ID!
+  owner: Bytes!
+  displayName: String!
+  imageUrl: String!
+}
+```
+
+Next, we generate the entities from the schema and check the squid builds:
+```bash
+make codegen
+make build
+```
+
+After that start the local database and generate the migrations from the generated entities:
+```bash
+make up
+make migration
+```
+The migration file will appear in `db/migrations`
+
+### Generate typings from ABI
+
+Copy the contract ABI (`Gravity.json`) to `src/abi` and run [EVM typegen](/develop-a-squid/typegen/squid-evm-typegen):
+```bash
+npx squid-evm-typegen --abi=src/abi/Gravity.json --output=src/abi/Gravity.ts
+```
+
+The generated access class will be generated in `src/abi/Gravity.ts`. In particular, it will contain the boilerplate to decode the logs and the topic definitions used at the next step. 
+
+### Subscribe to EVM logs
+
+The core of the indexing logic is implemented in `src/processor.ts`. This is where we define which EVM logs (events) our squid will process and the batch handler for them. Squids are configured directly in the code, unlike subgraphs which require the handlers and event to be defined in the manifest file.
+
+```ts file=src/processor.ts
+import { events } from "./abi/Gravity";
+
+const processor = new EvmBatchProcessor()
+  .setDataSource({
+    chain: process.env.ETHEREUM_MAINNET_WSS,
+    archive: 'https://eth-test.archive.subsquid.io',
+  })
+  .setBlockRange({ from: 6175243 })
+  .addLog('0x2E645469f354BB4F5c8a05B3b30A929361cf77eC', {
+    filter: [[
+      events['NewGravatar(uint256,address,string,string)'].topic,
+      events['UpdatedGravatar(uint256,address,string,string)'].topic,
+   ]],
+    data: {
+        evmLog: {
+            topics: true,
+            data: true,
+        },
+    } as const,
+});
+```
+
+In the snippet above we tell the squid processor to fetch logs emitted by the contract `0x2E645469f354BB4F5c8a05B3b30A929361cf77eC` that match of the specified topics. Note that the topic filter is a double array as required by the [selector specification](https://docs.ethers.io/v5/api/utils/abi/interface/#Interface--selectors). The configuration also specifies that the indexing should start from block `6175243` onwards (when the contract was deployed).
```

Comment:
Do you want to say anything specific about the data source? After all, they should change the `ETHEREUM_MAINNET_WSS` env variable, no?